app/javascript/application.js

```diff
@@ -1,3 +1,7 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
 import "@hotwired/turbo-rails";
 import "controllers";
+
+Turbo.StreamActions.redirect = function () {
+  Turbo.visit(this.target);
+};
```

Comment:
Custom stream action that can be used to trigger a "refresh" to the page after a successful form submission.  This is helpful because many of our forms are rendered in modals, which means for validations/errors, we need to leverage Turbo frames, but for successful submissions, we generally want a full refresh.  Since Turbo doesn't provide an easy native way to handle this behavior, this provides a low-overhead, simple workaround:

```ruby
redirect_target_url = request.referer || account_path(@entry.account)
        format.turbo_stream { render turbo_stream: turbo_stream.action(:redirect, redirect_target_url) }
```